client/src/common/components/AppLogo.scss

```diff
@@ -2,8 +2,11 @@
   display: flex;
   align-items: center;
 
-  &__icon {
+  span.app-logo__icon {
     height: 16px;
```

Comment:
Please use `constant-size` helper to avoid SVG sizes to be reduced below the expected size.